### PR TITLE
[rustfmt] introduce custom rustfmt configuration file for fuse

### PR DESCRIPTION
**Pull request by sundy-li**
## Summary

We must have a standard [rustfmt style](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md) for fuse-query,
this will be checked automatically and  save lots of time for the reviewers.

## Changelog

- Improvement

### CODE REVIEW

File: src/bin/fuse-query.rs [hunk 1]
```diff
@@ -2,14 +2,13 @@
...
 
-use log::info;
-
 use fuse_query::clusters::Cluster;
 use fuse_query::configs::Config;
 use fuse_query::metrics::MetricService;
 use fuse_query::rpcs::{HttpService, RpcService};
 use fuse_query::servers::MySQLHandler;
 use fuse_query::sessions::Session;
+use log::info;
```
Comment (BohuTANG) [inline src/bin/fuse-query.rs:11]: This is strange, why move here?

**Comment by BohuTANG** [comment]
This pr is very useful for the code style, thanks @sundy-li

File: src/bin/fuse-query.rs (hunk 1 above)
Comment (sundy-li) [reply src/bin/fuse-query.rs:11]: For `fuse-query.rs `,  crate `fuse_query` and `log` are all thirdparty crates.

Comment (BohuTANG) [reply src/bin/fuse-query.rs:11]: Got, I see

**Review by BohuTANG — APPROVED** [review]

**Comment by BohuTANG** [comment]
Cook, LGTM

**Pull request merged**